espnet2/gan_codec/encodec/seanet_st.py

```diff
@@ -0,0 +1,438 @@
+# Adapted from https://github.com/facebookresearch/encodec

```

Comment:
For adding speechtokenizer, could you please limit the change to either shared folder or speechtokenizer?

Reply:
Also, may I know what is the difference between this seanet and the existing one? If there is no significant changes, you can directly use the current seanet in the shared folder.